<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncluirDVD</class>
 <widget class="QDialog" name="IncluirDVD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>INCLUIR NOVO DVD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>261</width>
     <height>92</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>30</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Preço:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="input_dvd_preco"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Duração:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="input_dvd_nome"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="input_dvd_duracao"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nome:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IncluirDVD</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IncluirDVD</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
